napari/layers/base/base.py

```diff
@@ -1097,3 +1076,6 @@ def save(self, path: str, plugin: Optional[str] = None) -> List[str]:
         from ...plugins.io import save_layers
 
         return save_layers(path, [self], plugin=plugin)
+
+    def _on_selection(self, selected: bool):
```

Comment:
Ok, nice - I see why need this now. I think I might add comments like "would be nice to remove", i think will be cleaner if we don't need this anymore but it might take time